gradle/missing-javadoc.gradle

```diff
@@ -162,7 +162,6 @@ configure([
   project(":qa:os"),
   project(":qa:wildfly"),
   project(":rest-api-spec"),
-  project(":server"),
```

Comment:
This change is not very unclear: we do exactly what the same thing by extracting `configure(project(":server"))` into separate configuration block.

Reply:
It's starting to isolate the projects that will have `missingJavadoc` reenabled. To test this locally comment out `isExcluded = true` and uncomment the following two lines.

```
    // TODO: reenable after fixing missing javadocs
    // javadocMissingLevel = "class"
```

Reply:
Ah I see, thanks @nknize !